package/src/utils/define-integration.ts

```diff
@@ -69,8 +69,13 @@ export const defineIntegration = <
 						}),
 					addVitePlugin: (plugin) =>
 						addVitePlugin({ plugin, updateConfig: params.updateConfig }),
-					hasIntegration: (name) =>
-						hasIntegration({ name, config: params.config }),
+					hasIntegration: (other: string, position?: "before" | "after", relativeTo?: string) =>
```

Comment:
```suggestion
					hasIntegration: (_name: string, position?: "before" | "after", relativeTo?: string) =>
```
Make sure to update the call as well